nextflow.config

```
@@ -249,15 +254,19 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     docker {
         conda.enabled          = false
         docker.enabled         = true
+        docker.registry        = 'quay.io'
```

```suggestion
        // docker.registry        = 'quay.io'
```
Let's keep it to the old default until non-quay modules are updated (i.e. until this [#3344](https://github.com/nf-core/modules/pull/3344) PR is merged).